cpp/src/strings/case.cu

```diff
@@ -47,6 +48,13 @@ enum TwoPass
     ExecuteOp     ///< run the string operation
 };
 
+/**
+ * @brief Disable special case mappings.  This can be useful for situations when trying to
+ * regenerate the mapping hash table (where you might want to generate 'incorrect' results
+ * via an external verification tool).
+ */
+#define CUDF_ALLOW_SPECIAL_CASE_MAPPING
```

Comment:
I imagine it's extremely infrequent.   Development of cudf itself only.  What did you have in mind, template bool?

Reply:
just don't like the idea of maintaining untested code. I would prefer a different function that is properly tested so that when it is needed, it works.

Reply:
Thinking about this more, this was really just a thing that would be used when generating the hash table input data using the method I used (comparing 'bad' output against expected results).  A more general solution might be to generate this from the unicode data specification itself.   So this ends up being unnecessary - removing.